content/slides/intro-to-jupyterlab.qmd

```diff
@@ -105,6 +113,41 @@ pop-up in CryoCloud.
 
 ::::::
 
+## Running terminals and kernels ![](/_media/jupyterlab_running_terminals_kernels_icon.png){width="3%" style="vertical-align:middle"}
+:::::: {.columns}
+
+::: {.column}
+* Lists currently running Terminals and Kernels.
+
+* If one of these apps gets 'stuck', you may need to shut it down and
+  try again
+
+* If you accidentally close a terminal or notebook tab, you can re-access it by
+  clicking on its entry here.
+:::
+
+::: {.column}
+![JupyterLab Running Terminals and Kernels](/_media/jupyterlab_running_terminals_kernels.png)
+:::
+
+::::::
+
+:::{.notes}
+We will not be covering the usage of the final three side panel views.
+:::
+
+#
+
+![JupyterLab Launcher](/_media/jupyterlab_launcher.png)
+
+:::{.notes}
+Applications can be launched from the Launcher, opening a new tab/window for
+that application.
+
+To open a new Launcher, click the "+" tab, `File > New Launcher`, or the Blue
+`+` button in the file browser left side panel.
```

Comment:
We had a todo for showing how to open a jupyter notebook. I think talking about the launcher view and how new apps are opened from there will cover this (along with a live demo, of course).